Customer Support Procedures/LKE-v4.1-migration.md

```diff
@@ -0,0 +1,131 @@
+  - [Simplified log configuration](#simplified-log-configuration)
+  - [Security updates](#security-updates)
+- [Dropped Support section](#dropped-support-section)
+
+# Requirements
+
+Updating from Linkurious Enterprise v2.0 or older to v4.1 is not supported.
+If you are updating from Linkurious Enterprise v2.0 or older, please first update to v3.1 or v4.0 before updating to v4.1.
+
+We strongly recommend backing-up your SQL database and your installation folder before updating Linkurious Enterprise.
+
+# General major changes
+
+## Improved support for resources migration
+
+Universally unique identifiers (UUID) are 128-bit labels that look like `c7b30a93-207b-417e-b4f5-4f6e45818595`.
+A UUID uniquely identifies a resource, no matter its kind and the Linkurious Enterprise instance it resides in.
+
+UUIDs have been added to Linkurious enterprise's resources:
+- Custom groups
+- Static queries & query templates
+- Custom actions
+- Node grouping rules
+- Spaces
+- Alerts & alert folders
+
+
+***Impacted clients:***
+
+Everyone using the Configuration Migration Plugin.
+
+***Benefits:***
+
+This solves issues caused by renamed resources, allowing more reliable synchronizations between Linkurious Enterprise instances.
+
+***Impacts:***
+
+Static queries and query templates can be referred to by their identifier in various parts of Linkurious Enterprise:
+- In [Custom actions][1]
+- In [Deep links][2]
+- To populate the [Guest mode workspace][3]
+- In the [Data-table Plugin][4]
+
+In each of these references, the query ID should be replaced by the associated UUID,
+to ensure the resource referencing the query can be synchronized to another Linkurious Enterprise instance without inconsistencies.
+
+You can get the list of all your queries with their ID and and UUID using the [Get queries API][5].
+
+## Resources ownership
+
+In Linkurious 4.1, we aim to standardize and improve user rights to facilitate onboarding and offboarding of new employees,
+as well as giving Linkurious administrators greater freedom to manage access restrictions to the application's functionalities.
+
+A new administrator right has been added for queries, custom actions and alerts.
+This right allows the user to edit and delete any asset created by other Linkurious users.
+We have also added node group rule creation rights, to harmonize rights.
+A node group rule can therefore be modified and deleted by the creator
+or a user with the `Manage, edit read/write queries & run` right level.
+
+***Impacted clients:***
+
+All Linkurious clients using queries, custom actions, alerts or node grouping.
+
+***Benefits:***
+
+Rights management has been harmonized between queries, custom actions, alerts and node grouping.
+A user with sufficient rights can edit and delete any element created by other users.
+Migration of assets created between different instances (using the Configuration Migration Plugin) has also been facilitated.
+
+***Impacts:***
```

Comment:
Here we can also add that any custom group will get the "Can create and apply node grouping rules" if u think it's needed

Reply:
@maxime-allex I'll let you handle that change